assets/src/styles/editorStyle.scss

```diff
@@ -9,6 +9,8 @@
 @import "master-theme/assets/src/scss/base/functions";
 @import "master-theme/assets/src/scss/base/mixins";
 @import "master-theme/assets/src/scss/base/fonts";
+@import "master-theme/assets/src/scss/components/buttons";
```

Comment:
Master theme now has its own `editorStyle.scss` file, maybe it's better to import it there?